scipy/stats/_stats_py.py

```diff
@@ -6963,6 +6965,10 @@ def _unequal_var_ttest_denom(v1, n1, v2, n2):
 
 
 def _equal_var_ttest_denom(v1, n1, v2, n2):
+    # null variance in case there is a single observation
+    v1 = 0 if n1 == 1 else v1
```

Comment:
Not sure but maybe this being 0 is instead of NaN is causing the divide by 0 error. That can be fixed by using `np.divide` instead or by making an argument of the division a NumPy 0.

Reply:
I added `np.divide` and it did solve an issue.